cardano-db-tool/src/Cardano/DbTool/Validate/TxAccounting.hs

```diff
@@ -45,8 +45,8 @@ import qualified System.Random as Random
 
 {- HLINT ignore "Fuse on/on" -}
```

Comment:
I know you didn't add this, but since you're already in there, you may as well remove this HLINT hint, since it is ignored globally

Reply:
I'll do one better and remove them all as it seems it's in 11 places 😗